charts/vela-core/templates/kubevela-controller.yaml

```
@@ -122,6 +122,7 @@ metadata:
   name: {{ include "kubevela.fullname" . }}
   namespace: {{ .Release.Namespace }}
   labels:
+    controller.oam.dev/name: application
```

```suggestion
    controller.oam.dev/name: vela-core
```

You need to check if user upgrade to this version have this label.

---

Yes.